ern-orchestrator/src/runMiniApp.ts

```diff
@@ -168,13 +171,18 @@ export async function runMiniApp(
   })
 
   if (platform === 'android') {
+    extra = extra || {}
+    extra.androidConfig = {
+      ...(extra && extra.androidConfig),
+      artifactId: `runner-ern-container-${entryMiniAppName.toLowerCase()}`,
+      groupId: 'com.walmartlabs.ern',
```

Comment:
minor -> extract to a constant (same string in line no. 235)

Reply:
Can these string references go to `ern-core`? In which case `ern-orchestrator` & `ern-runner-gen-android` both have reference to it? In the future decide to change, it's central point where you may change.

Reply:
@krunalsshah I will take this in a separate PR since there are lot of strings that can make it's way to a constant in the same file or to the core. Will address it in general.